source/MBF.SAMD51.GPIO.pas

```diff
@@ -98,6 +121,69 @@         TArduinoPin = record
         A4 = TNativePin.PB8;   A5 = TNativePin.PB9;
       end;
     {$endif}
+    {$ifdef fpc_mcu_wio_terminal}
+      type
+        TArduinoPin = record
```

Comment:
Defining only one Arduinopin does not maker sense, either a board has Arduino compatible Headers or do not define this at all

Reply:
I did this to get the more general Blinky example running out-of-the-box on the Wio-terminal. So, Blinky can work and blink the single user LED available on Wio Terminal.